packages/core/core/src/factories/useForgotPasswordFactory.ts

```diff
@@ -0,0 +1,70 @@
+import { CustomQuery, Context, FactoryParams, UseForgotPasswordErrors, UseForgotPassword } from '../types';
+import { Ref, computed } from '@vue/composition-api';
+import { sharedRef, Logger, configureFactoryParams } from '../utils';
+
+interface IChangePasswordParams {
```

Comment:
Kind of new naming convention in our project. Maybe we should discuss it on dev-general? Wdyt?

Reply:
Good idea! I will create discussion for that.

Reply:
We decided not to use that after the discussion.